docs/detections/detections-ui-exceptions.asciidoc

```diff
@@ -115,6 +126,10 @@ NOTE: When using a list, all exception statements must use `is in list` and
 is only available when adding exceptions via the Alerts table.
 * _Close all alerts that match this exception, including alerts generated by other rules_:
 Closes all alerts that match the exception's conditions.
++
+IMPORTANT: When you select to close all alerts that meet the exception's
+criteria, all matching alerts are closed, *including* alerts generated by other
+rules.
```

Comment:
This is actually no longer true. We updated this in 7.10 to only close alerts generated by the rule in question. https://github.com/elastic/kibana/pull/77402

![image](https://user-images.githubusercontent.com/616158/98723198-102ab900-2360-11eb-9425-3d3634386378.png)